scripts/jest/environment.prod.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+global.__DEV__ = false;
```

Comment:
do we need to set NODE_ENV to 'production' too for fbjs to work? (caution that in the past this made the transforms fail for me because the transforms read NODE_ENV and expect it to be 'test'. might need to set it to 'test' and back before/after the babel transform in the jest preprocessor.)